CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.3.0]
+### Added
+- More exports to index.ts ([#86](https://github.com/MetaMask/auto-changelog/pull/86))
+  - Specifically, the `Changelog` class, `createEmptyChangelog`, and `parseChangelog`.
+
+### Fixed
+- Ensure tags are retrieved for merge commits ([#87](https://github.com/MetaMask/auto-changelog/pull/87))
+  - Previously, we passed the path to the project root directory as a param to an invocation of `git rev-list`, which excluded any merge commits and therefore also their tags.
```

Comment:
Hm, I don't think that's quite right. I discovered this bug while using this package with a non-monorepo, and the problem with merge commits is correct. The monorepo-specific problem would be that we retrieved a package-specific rather than a repository-wide git tag. In `updateChangelog`, we were already computing a range of commits, it's just that the start of the range was wrong. See here:

https://github.com/MetaMask/auto-changelog/blob/main/src/update-changelog.ts/#L153-L159

How about this?

```suggestion
- Correctly calculate the most recent git tag ([#87](https://github.com/MetaMask/auto-changelog/pull/87)) ([#87](https://github.com/MetaMask/auto-changelog/pull/87))
  - Previously, we passed the path to the project root directory as a parameter to an invocation of `git rev-list`. For all repositories, this caused tags belonging to merge commits to be excluded. For monorepos, this also caused tags belonging to commits that didn't change any files belonging to a specific package / workspace to be excluded.
```

Reply:
Good point on it not being specific to monorepos. But the problem wasn't with merge commits specifically. Merge commits just happen to be a common example because they are empty more often than non-merge commits are, so it's pretty likely they wouldn't touch the path we're filtering by. But it applies to regular commits too.

e.g. lets say that in a monorepo with package A, the the most recent commit in v1 was a top-level README update. But the commit before that was the version bump for package A. We would have excluded v1 was a "most recent tag" candidate for package A despite the fact that package A was released as V1.

Reply:
Oh I see - your last sentence addressed this :facepalm: Should have read until the end